lib/src/http/managers/message_manager.dart

```diff
@@ -325,10 +325,13 @@ class MessageManager extends Manager<Message> {
   }
 
   MessageInteractionMetadata parseMessageInteractionMetadata(Map<String, Object?> raw) {
+    final userId = Snowflake.parse((raw['user'] as Map<String, Object>)['id']!);
+
     return MessageInteractionMetadata(
       id: Snowflake.parse(raw['id']!),
       type: InteractionType(raw['type'] as int),
-      userId: Snowflake.parse(raw['user_id']!),
+      userId: userId,
+      user: client.users[userId],
```

Comment:
Docs say this is a full user object - is it not?

Reply:
It's a partial user, with some missing fields.